siriuspy/siriuspy/devices/blctrl.py

```diff
@@ -0,0 +1,85 @@
+"""Beamline Control."""
+
+from .device import Device as _Device
```

Comment:
shouldn't it be derivated from DeviceNC?

Reply:
done